src/parquet/statistics.cc

```diff
@@ -96,6 +96,67 @@ void TypedRowGroupStatistics<DType>::Reset() {
   has_min_max_ = false;
 }
 
+template <typename T>
+inline int getValueBeginOffset(const T* values, int64_t count) {
```

Comment:
Can we use compliant naming in these new functions (function names should be capitalized)?